src/Microsoft.AspNet.Mvc.ApiExplorer/DefaultApiDescriptionProvider.cs

```diff
@@ -372,6 +372,13 @@ private Type GetDeclaredReturnType(ControllerActionDescriptor action)
             }
         }
 
+        private static Type GetTaskInnerTypeOrNull([NotNull] Type type)
```

Comment:
nitpick: no need for `[NotNull]` on private methods. If you're concerned about a high likelihood of this happening, use `Debug.Assert`